backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -196,6 +196,52 @@ private async Task UpdateOrgMemberRole(LexBoxDbContext dbContext, Organization o
         await dbContext.SaveChangesAsync();
     }
 
+    public record BulkAddOrgMembersInput(Guid OrgId, string[] Usernames, OrgRole Role);
+    public record OrgMemberRole(string Username, OrgRole Role);
+    public record BulkAddOrgMembersResult(List<OrgMemberRole> AddedMembers, List<OrgMemberRole> NotFoundMembers, List<OrgMemberRole> ExistingMembers);
+
+    [Error<NotFoundException>]
+    [Error<DbError>]
+    [AdminRequired]
+    [UseMutationConvention]
+    public async Task<BulkAddOrgMembersResult> BulkAddOrgMembers(
+        BulkAddOrgMembersInput input,
+        LexBoxDbContext dbContext)
+    {
```

Comment:
This is naturally quite similar to the BulkAddProjectMembers mutation, which is not surprising since it was created by copying the BulkAddProjectMembers method, changing "project" to "org" everywhere, and then removing the irrelevant code such as inviting users not found. The result is different enough, however, that I don't think the two should be refactored to extract common code, as the refactored version would be harder to read than the copied version. (Plus, we actually *want* these two methods to be able to diverge more in the future).

Same comment applies to the UI: the BulkAddOrgMembers component is largely copied from BulkAddProjectMembers, but refactoring the two to share common code would not be worth it, as we want them to be able to diverge. (I won't repeat the comment on the UI file).